.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "google",
+    "prettier"
+  ],
+  "env": {
+    "browser": true,
+    "commonjs": true
+  },
+  "plugins": ["@typescript-eslint"],
+  "rules": {
+    "require-jsdoc": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "valid-jsdoc": "warn",
+    "@typescript-eslint/no-unused-vars": "off", // overlaps with standard eslint no-unused-vars.
```

Comment:
We should also enable no-floating-promise rule: https://typescript-eslint.io/rules/no-floating-promises
Maybe in a follow up PR.

We could also try enabling strict checks and see whether it adds a lot of annoying errors.
https://typescript-eslint.io/docs/linting/configs#strict

Reply:
+1 to doing it after this goes in